engines/stark/resources/textureset.cpp

```diff
@@ -117,7 +117,7 @@ void TextureSet::extractArchive() {
 }
 
 Gfx::TextureSet *TextureSet::readOverrideDdsArchive() {
-	Common::String archiveName = _filename + ".zip";
+Common::String archiveName = _filename + ".zip";
```

Comment:
Why? Please revert this

Reply:
Huh, this is weird, I didn't actually modify that myself (doesn't show up as a diff in VS either). Any idea what might be causing this?